wasm/src/bindings_c2d.cpp

```diff
@@ -20,410 +21,444 @@
-      .value("screen", rive::BlendMode::screen)
-      .value("overlay", rive::BlendMode::overlay)
-      .value("darken", rive::BlendMode::darken)
-      .value("lighten", rive::BlendMode::lighten)
-      .value("colorDodge", rive::BlendMode::colorDodge)
-      .value("colorBurn", rive::BlendMode::colorBurn)
-      .value("hardLight", rive::BlendMode::hardLight)
-      .value("softLight", rive::BlendMode::softLight)
-      .value("difference", rive::BlendMode::difference)
-      .value("exclusion", rive::BlendMode::exclusion)
-      .value("multiply", rive::BlendMode::multiply)
-      .value("hue", rive::BlendMode::hue)
-      .value("saturation", rive::BlendMode::saturation)
-      .value("color", rive::BlendMode::color)
-      .value("luminosity", rive::BlendMode::luminosity);
-
-  class_<rive::rcp<rive::RenderShader>>("RenderShader");
-
-  class_<rive::RenderPaint>("RenderPaint")
-      .function("color", &RenderPaintWrapper::color, pure_virtual(),
-                allow_raw_pointers())
-
-      .function("style", &RenderPaintWrapper::style, pure_virtual(),
-                allow_raw_pointers())
-      .function("thickness", &RenderPaintWrapper::thickness, pure_virtual(),
-                allow_raw_pointers())
-      .function("join", &RenderPaintWrapper::join, pure_virtual(),
-                allow_raw_pointers())
-      .function("cap", &RenderPaintWrapper::cap, pure_virtual(),
-                allow_raw_pointers())
-      .function("blendMode", &RenderPaintWrapper::blendMode, pure_virtual(),
-                allow_raw_pointers())
-      .function("shader", &RenderPaintWrapper::shader, pure_virtual(),
-                allow_raw_pointers())
-      .allow_subclass<RenderPaintWrapper>("RenderPaintWrapper");
-
-  class_<rive::RenderImage>("RenderImage")
-//      .function("decode", &RenderImageWrapper::decode, pure_virtual(), allow_raw_pointers())
-      .function("size", &RenderImageWrapper::size)
-      .allow_subclass<RenderImageWrapper>("RenderImageWrapper");
-
+EMSCRIPTEN_BINDINGS(RiveWASM_C2D)
+{
+    class_<rive::Renderer>("Renderer")
+        .function("save", &RendererWrapper::save, pure_virtual(), allow_raw_pointers())
+        .function("restore", &RendererWrapper::restore, pure_virtual(), allow_raw_pointers())
+        .function("transform", &RendererWrapper::transform, pure_virtual(), allow_raw_pointers())
+        .function("drawPath", &RendererWrapper::drawPath, pure_virtual(), allow_raw_pointers())
+        .function("clipPath", &RendererWrapper::clipPath, pure_virtual(), allow_raw_pointers())
+        .function("align",
+                  optional_override([](rive::Renderer& self,
+                                       rive::Fit fit,
+                                       JsAlignment alignment,
+                                       const rive::AABB& frame,
+                                       const rive::AABB& content) {
+                      self.align(fit, convertAlignment(alignment), frame, content);
```

Comment:
Same deal as other comment, override the default align with one that converts the simple JS Alignment enum to our actual rive::Alignment version.